samples/snippets/csharp/VS_Snippets_VBCSharp/CsProgGuideTypes/CS/Class1.cs

```diff
@@ -555,6 +555,17 @@ static void Main()
 
             // Output: 200.0056
             //</snippet39>
+
+            //<snippet40>
```

Comment:
⚠️ This is a wrong change
There is already an existing snippet40, snippet tags should be unique.